CHANGELOG

```diff
@@ -1,2 +1,77 @@
-1.1.2
-- Changed to nested module style see: https://github.com/bitcrowd/rubocop-bitcrowd/pull/4
+# Rubocop Bitcrowd Changelog
```

Comment:
The file is missing a `.md` extension. Would add it so that Github can e.g. render it as Markdown.